src/main/java/seedu/address/model/Model.java

```diff
@@ -90,12 +90,18 @@ public interface Model {
     void viewNote(Person person);
 
     /**
-     * Replaces person list with the specified person list
+     * Sorts person list with the specified person list
      *
-     * @return
+     * @return int to throw error for no contact with list in the
      */
     int sortFilteredPersons();
 
+    /**
+     * Sorts the person by Name
+     *
+     */
+    void sortByName();
+
```

Comment:
Maybe can remove this since  I added the ObservableList<Person> getSortedList() in case they consider this as a duplicated method.

Reply:
Ohh...yes! I forgot to remove it. Will just do that